### PR TITLE
Dimension documentation

### LINKED ISSUE #27: cusfsim.case.Dimension arguments are undocumented
The arguments to `cusfsim.case.Dimension` arguments are undocumented. It's not clear, for example, what order they are in or what they represent.

Add some documentation which lists them.

### PR DESCRIPTION
**Pull request by jb803**
Adds some documentation to the Dimension class. This should resolve #27

### CODE REVIEW

File: cusfsim/case.py [hunk 1]
```diff
@@ -81,7 +114,7 @@ class Dimension(PFDataStructs.Dimension):
     >>> repr(d)
     'cusfsim.case.Dimension(0, 1, -2, 0, 0, 0, 0)'
 
-    It supports indexing and the sequence property
+    The class also supports indexing and the sequence property
```
Comment (rjw57) [inline cusfsim/case.py]: I've just noticed that this is a thinko. It should be "and the sequence protocol". As it is, I'll fix in a separate PR.

**Comment by rjw57** [comment]
:+1: will merge if tests pass

**Comment by rjw57** [comment]
@jb803 Can you hit the "update branch" button? Thanks.

**Comment by jb803** [comment]
Branch updated :)

**Pull request merged**